### PR TITLE
Help user install plugins

### LINKED ISSUE #133: Print instructions or automatically move binary to plugin dir
When we do a validator join, we should copy the VM to the plugin dir or print instructions to do so.

### PR DESCRIPTION
**Pull request by dasconnor**
Added the ability to print instructions for installing the VM binary and complemented that with functionality to do it automatically.

Closes #133

### CODE REVIEW

File: pkg/constants/constants.go [hunk 1]
```diff
@@ -53,4 +53,6 @@ const (
 
 	DefaultConfigFileName = ".avalanche-cli"
 	DefaultConfigFileType = "json"
+
+	DefaultPluginDir = "plugins"
```
Comment (felipemadero) [inline pkg/constants/constants.go:57]: really this is not going to be the plugin dir used by avalanchego, probably the name DefaultPluginDir is misleaving

File: pkg/binutils/binaries.go [hunk 2]
```diff
@@ -29,6 +29,7 @@ var (
 
 type PluginBinaryDownloader interface {
 	Download(vmIDs map[string]struct{}, pluginDir, binDir string) error
+	DownloadVM(vmID string, pluginDir, binDir string) error
```
Comment (felipemadero) [inline pkg/binutils/binaries.go:32]: why not use Download?

File: pkg/application/app.go [hunk 3]
```diff
@@ -68,6 +68,10 @@ func (app *Avalanche) GetKeyDir() string {
 	return filepath.Join(app.baseDir, constants.KeyDir)
 }
 
+func (app *Avalanche) GetPluginDir() string {
```
Comment (felipemadero) [inline pkg/application/app.go:71]: GetPluginDir may be misleaving as avalanchego is not going to use this directory

File: pkg/constants/constants.go (hunk 1 above)
Comment (dasconnor) [reply pkg/constants/constants.go]: this is a good point, I moved the referenced plugin dir to `/tmp`

File: pkg/binutils/binaries.go (hunk 2 above)
Comment (dasconnor) [reply pkg/binutils/binaries.go:32]: Download has a lot of additional complexity. Honestly, I had a lot of confusion trying to figure out how to use it. DownloadVM was much simpler and already public, just needed to be added to the interface.

File: pkg/application/app.go (hunk 3 above)
Comment (dasconnor) [reply pkg/application/app.go]: renamed

**Review by felipemadero — APPROVED** [review]
LGTM

**Pull request merged**